### ISSUE TITLE
Accessibility checks for v5

### ISSUE THREAD

**Issue by ffoodd** [Orange-OpenSource/Orange-Boosted-Bootstrap#296]
Whether to use Bootstrap features or not? Some of them should be tested against our accessibility requirements…

Here is a non exhaustive list:

- [x] Forms:
  - color picker → https://twbs-bootstrap.netlify.com/docs/4.3/forms/form-control/#color
  - datalist → https://twbs-bootstrap.netlify.com/docs/4.3/forms/form-control/#datalists
  - input groups → https://twbs-bootstrap.netlify.com//docs/4.3/forms/input-group/
  - checks without labels →https://twbs-bootstrap.netlify.com/docs/4.3/forms/checks/#without-labels
  - validation messages → https://twbs-bootstrap.netlify.com/docs/4.3/forms/validation/
- [x] Carousel → https://twbs-bootstrap.netlify.com/docs/4.3/components/carousel/
- [x] Modal→ https://twbs-bootstrap.netlify.com/docs/4.3/components/modal/
- [x] Popovers → https://twbs-bootstrap.netlify.com/docs/4.3/components/popovers/
- [x] Tooltips → https://twbs-bootstrap.netlify.com/docs/4.3/components/tooltips/

Please take some time to read documentation and examples carefully, to ensure we deliver components at the state-of-the-art from the beginning.

**Comment by ffoodd** [comment]
FWIW since I'm now part of Bootstrap's team, I think if we have any suggestions to improve accessibility in some of those components we should reverse them directly :innocent:

**Comment by ffoodd** [comment]
Carousels may benefit from [WAI ARIA carousel best practices](https://www.w3.org/TR/wai-aria-practices-1.1/#carousel). To read carefully :)

**Comment by ffoodd** [comment]
@Aniort tested all the things and here are some comments:
- Color picker works great, except [a bug in Firefox where opening picker with keyboard does not send focus to the dialog](https://bugzilla.mozilla.org/show_bug.cgi?id=1526820)
- `datalist` isn't announced at all so isn't sufficient on its own. Makes #260 even more important!
- Tooltips and Popovers works fine, but when containing interactive elements, those aren't reachable (because they're at the end of the DOM and neither `tabindex` nor `focus` is handled).
- Carousels might be improved: 
  - [WAI ARIA carousel best practices](https://www.w3.org/TR/wai-aria-practices-1.1/#carousel)
  - [WAI Carousel Tutorial](https://www.w3.org/WAI/tutorials/carousels/)
  - [Example demo on CodePen by Adilade](https://codepen.io/Adilade/pen/RmvpWK)
